ansible_role/playbooks/operator.yml

```diff
@@ -5,9 +5,17 @@
   become: false
   gather_facts: false
   connection: local
-  roles:
-  - role: automation-broker
   vars:
     state: present
     broker_name: "{{ meta.name }}"
     broker_namespace: "{{ meta.namespace }}"
+    broker_image: "{{ lookup('env', 'IMAGE') }}"
+  tasks:
+    - assert:
+        that:
+          - broker_name is defined
+          - broker_namespace is defined
+          - broker_image is defined
+        msg: broker_name broker_namespace broker_image must be defined
```

Comment:
I would change this to indicate that the `IMAGE` environment variable must provide the broker's image name, since `broker_image` is an internal variable and the env var is the public interface.

Reply:
Done